src/marqo/tensor_search/api.py

```diff
@@ -172,7 +172,7 @@ def add_or_replace_documents(
         request: Request,
         body: typing.Union[AddDocsBodyParams, List[Dict]],
         index_name: str,
-        refresh: bool = True,
+        refresh: bool = False,
```

Comment:
In this case, is it possible to move the default for this value into the `AddDocsParams`?

Reply:
(but no need to consolidate other add_docs params)

Reply:
definitely

Reply:
Actually, the only way I see this happening is by setting the default for the `refresh` parameter here to `None`, passing that to the orchestrator, then handling logic for `auto_refresh is None` inside `addDocsParams`. It feels a bit roundabout.

Another way to do it is to add logic to the API and the orchestrator to not send the `refresh` parameter at all if not provided by the API call, so that `addDocsParams` fills it in by itself, but that goes against our principle of having less logic in the API.